types/tronweb/index.d.ts

```diff
@@ -0,0 +1,115 @@
+    getTransaction(transactionID: any, callback?: any): Promise<any>;
+    getConfirmedTransaction(transactionID: any, callback?: any): Promise<any>;
+    getTransactionInfo(transactionID: any, callback?: any): Promise<any>;
+    getTransactionsToAddress(address: any, limit: number, offset: number, callback?: any): Promise<any>;
+    getTransactionsFromAddress(address: any, limit: number, offset: number, callback?: any): Promise<any>;
+    getTransactionsRelated(address: any, direction: any, limit: number, offset: number, callback?: any): Promise<any>;
+    getAccount(address: any, callback?: any): Promise<any>;
+    getBalance(address: any, callback?: any): Promise<any>;
+    getUnconfirmedAccount(address: any, callback?: any): Promise<any>;
+    getUnconfirmedBalance(address: any, callback?: any): Promise<any>;
+    getBandwidth(address: any, callback?: any): Promise<any>;
+    getTokensIssuedByAddress(address: any, callback?: any): Promise<any>;
+    getTokenFromID(tokenID: any, callback?: any): Promise<any>;
+    listNodes(callback?: any): Promise<any>;
+    getBlockRange(start: number, end: number, callback?: any): Promise<any>;
+    listSuperRepresentatives(callback?: any): Promise<any>;
+    listTokens(limit?: number, offset?: number, callback?: any): Promise<any>;
+    timeUntilNextVoteCycle(callback?: any): Promise<any>;
+    getContract(contractAddress: any, callback?: any): Promise<any>;
+    verifyMessage(message: any, signature: any, address: any, useTronHeader: any, callback?: any): Promise<any>;
+    sign(transaction: any, privateKey: any, useTronHeader: boolean, callback?: any): Promise<any>;
+    sendRawTransaction(signedTransaction: any, options: any, callback?: any): Promise<any>;
+    sendTransaction(to: any, amount: any, options: any, callback?: any): Promise<any>;
+    sendToken(to: any, amount: any, tokenID: any, options: any, callback?: any): Promise<any>;
+    freezeBalance(amount: any, duration: number, resource: string, options: any, callback?: any): Promise<any>;
+    unfreezeBalance(resource: string, options: any, callback?: any): Promise<any>;
+    updateAccount(accountName: string, options: any,  callback?: any): Promise<any>;
+    signMessage(...args: any[]): Promise<any>;
+    sendAsset(...args: any[]): Promise<any>;
+    send(...args: any[]): Promise<any>;
+    sendTrx(...args: any[]): Promise<any>;
+    broadcast(...args: any[]): Promise<any>;
+    signTransaction(...args: any[]): Promise<any>;
+    getProposal(proposalID: any, callback?: any): Promise<any>;
+    listProposals(callback: any): Promise<any>;
+    getChainParameters(callback: any): Promise<any>;
+    getAccountResources(address: any, callback?: any): Promise<any>;
+    getExchangeByID(exchangeID: any, callback?: any): Promise<any>;
+    listExchanges(callback?: any): Promise<any>;
+    listExchangesPaginated(limit: number, offset: number, callback?: any): Promise<any>;
+}
+
```

Comment:
The JS library you're writing definitions for uses CommonJS style modules and exports a class. You should use [this template](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html). Specifically, the `export <foo>` syntax in this file will need to be replaces with one `export =` and the other exports must be namespaced (this is what the template is doing).